resources/assets/js/app.js

```diff
@@ -277,6 +277,28 @@ $(function() {
         $this.next('.group-items').toggleClass('hide');
     });
 
+    $('.select-group').on('click', function () {
+        var $this = $(this);
+        var $parentGroup = $this.closest('ul.list-group');
```

Comment:
Since you're only referencing `$this` once, you may as well just make it `$(this).closest(...)`.